cmd/kubelet/app/server.go

```diff
@@ -926,7 +926,11 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 	}
 
 	// If systemd is used, notify it that we have started
-	go daemon.SdNotify(false, "READY=1")
+	go func() {
+		if _, err := daemon.SdNotify(false, daemon.SdNotifyReady); err != nil {
+			klog.ErrorS(err, "Failed to send systemd daemon successful start message")
+		}
+	}()
```

Comment:
done, and i will thinking about the other scene in next pr. thanks for your review.

Reply:
It would be good to understand two things before going forward:
- why genericapiserver used `READY=1\n` and if it's ok to change it to `READY=1` as you did
- why kubelet runs this in a goroutine, but API server doesn't and which way is better

Reply:
hi @bart0sh, thanks for your suggestions. 
1. why genericapiserver used `READY=1\n`
    the `daemon` package implements the [sd_notify protocol](https://www.freedesktop.org/software/systemd/man/sd_notify.html#Description). according to the documentation, systemd parses encoded data in an environment-block-like string, such as `READY=1\nSTATUS=Processing requests…\n`. When a message contains multiple key-value pairs, they must be separated with `\n`. if the message has only one key-value pair, both `READY=1` and `READY=1\n` are acceptable.    

based on the description, i conducted some tests.    
first, i added logs to print the result of the notify,
```go
// staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go
b, err := systemd.SdNotify(true, systemd.SdNotifyReady)
if err != nil {
	klog.Errorf("Unable to send systemd daemon successful start message: %v\n", err)
}
klog.Infof("==notify result %v", b)
```
next, i modified the `kube-apiserver.service` file to change its type as follows
```
[Service]
Type=notify
NotifyAccess=main
```
finally, the status of `kube-apiserver.service` is correct, and the log as follows
```
Sep 26 16:33:22 VM-20-13-centos kube-apiserver[237565]: I0926 16:33:22.591046  237565 secure_serving.go:213] Serving securely on [::]:6443
Sep 26 16:33:22 VM-20-13-centos kube-apiserver[237565]: I0926 16:33:22.591174  237565 genericapiserver.go:757] ==notify result true
```

2. the diff of kubelet and kube-apiserver about notify
    i believe it might be better not to use goroutines. SdNotify communicates with systemd via local unix socket, which are typically non-blocking, so introducing goroutines doesn’t seem particularly necessary.

   regarding this, could you give me some better advice, tks~  :)